addons/l10n_my_edi_extended/models/account_edi_xml_ubl_my.py

```diff
@@ -20,6 +20,8 @@ def _export_invoice_vals(self, invoice):
                 'industry_classification_code_attrs': {'name': invoice.partner_id.commercial_partner_id.l10n_my_edi_industrial_classification.name},
                 'industry_classification_code': invoice.partner_id.commercial_partner_id.l10n_my_edi_industrial_classification.code,
             })
+            # Self-billed invoices must use the Bill Reference is set; as we want to see appear the supplier bill number not Odoo's
```

Comment:
Nitpick: I find the comment a bit complicated
```suggestion
            # Self-billed invoices must use the number given by the supplier
```